pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -243,39 +246,38 @@ private SuccessResponse addSchema(Schema schema, boolean override) {
       return new SuccessResponse(schemaName + " successfully added");
     } catch (Exception e) {
       _controllerMetrics.addMeteredGlobalValue(ControllerMeter.CONTROLLER_SCHEMA_UPLOAD_ERROR, 1L);
-      throw new ControllerApplicationException(LOGGER,
-          String.format("Failed to add new schema %s.", schemaName), Response.Status.INTERNAL_SERVER_ERROR,
-          e);
+      throw new ControllerApplicationException(LOGGER, String.format("Failed to add new schema %s.", schemaName),
```

Comment:
Won't this end up throwing 5xx if schema update failed due to backward incompatibility?

Reply:
This is auto-formatting change under the `addSchema()` function. We will throw 400 (BAD_REQUEST) error when users making backward incompatible schema update (line 292).